.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: npm build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request: {}
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      - name: Publish to npm
+        run: npm publish --tag next
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.npm_token}}
```

Comment:
I suppose that the token is supposed to be added to https://github.com/kaitai-io/kaitai_struct_javascript_runtime/settings/secrets for the build to work?

Reply:
Just added. It will be only used on non-PR builds, though.

Reply:
> Just added. It will be only used on non-PR builds, though

Yeah, makes sense, we don't want a PR from a random person to be immediately published as a official package 🙂